platform/android/src/test/http_file_source_test_stub.cpp

```diff
@@ -7,7 +7,7 @@ namespace mbgl {
 
 class HTTPFileSource::Impl {};
 
-HTTPFileSource::HTTPFileSource(const ResourceOptions& /*options*/)
+HTTPFileSource::HTTPFileSource(const ResourceOptions& /*options*/, const ClientOptions& /*options*/)
```

Comment:
I guess it does not really matter, but you could replace options with resourceOptions and clientOptions...

Reply:
Done.